lib/node_modules/@stdlib/complex/parse-float64/README.md

```diff
@@ -0,0 +1,142 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2024 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# parseComplex128
+
+> Parse a string representation of a 128-bit [complex number][@stdlib/complex/float64].
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<!-- Package usage documentation. -->
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var parseComplex128 = require( '@stdlib/complex/parse-float64' );
+```
+
+#### parseComplex128( str )
+
+Parses a string representation of a 128-bit [complex number][@stdlib/complex/float64]. Throws an error if the string is not recognized as a complex number.
+
+
+```javascript
+var str = '5 + 3j';
+
+try {
+    var z = parseComplex128( str );
+    // returns <Complex128>
+} catch (error) {
+    console.error('An error occurred:', error.message);
+}
+```
+
+For details on the string format, see [Complex128][@stdlib/complex/float64].
+
+</section>
+
+<!-- /.usage -->
+
+<!-- Package usage notes. Make sure to keep an empty line after the `section` element and another before the `/section` close. -->
+
+<section class="notes">
+
+</section>
+
+<!-- /.notes -->
+
+<!-- Package usage examples. -->
+
+<section class="examples">
+
+## Examples
+
+<!-- eslint no-undef: "error" -->
+
+```javascript
+var Complex128 = require( '@stdlib/complex/float64' );
+var parseComplex128 = require( '@stdlib/complex/parse-float64' );
+
+var str = '5 + 3j';
```

Comment:
This example is essentially the same as the example above. Perhaps we can show some different examples?